### PR TITLE
Update Remove-ADGroupMember.md

### PR DESCRIPTION
**Pull request by HerbertMauerer**
add new parameter -DisablePermissiveModify for bug Bug 27810755

### CODE REVIEW

File: docset/windows/addsadministration/Remove-ADGroupMember.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 ms.mktglfcycl: manage
 ms.sitesec: library
-ms.author: v-kaunu
+ms.author: jaiello
```
Comment (get-itips) [inline docset/windows/addsadministration/Remove-ADGroupMember.md:4]: Same comment as #2229 about changing author metadata

File: docset/windows/addsadministration/Remove-ADGroupMember.md [hunk 2]
```diff
@@ -307,6 +307,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisablePermissiveModify
+Group membership updates use permissive modify by default. This suppresses an error when removing a member that is not member of the group.
+When this parameter is used, an error “The specified account name is not a member of the group” is returned.
+
+This parameter is available in Windows Server 2019 with the September 2020 Updates.
```
Comment (dariomws) [inline docset/windows/addsadministration/Remove-ADGroupMember.md:314]:
```suggestion
### -DisablePermissiveModify
Group membership updates use permissive modify by default. This suppresses an error when removing a member that is not a member of the group.
When this parameter is used, an error “The specified account name is not a member of the group” is returned.

This parameter is available in Windows Server 2019 with the September 2020 Updates.
```

**Review by dariomws — COMMENTED** [review]
Agree with @get-itips and added a small suggestion.

Thanks!

**Review by tseward — APPROVED** [review]

**Comment by HerbertMauerer** [comment]
Hello,

the old value gave me an error as the alias is invalid. At least the line gave me an error on previous PRs.
From the attempt 23rd of October:
docset/windows/addsadministration/Add-ADGroupMember.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FHerbertMauerer%2Fwindows-powershell-docs%2Fblob%2Fpatch-1%2Fdocset%2Fwindows%2Faddsadministration%2FAdd-ADGroupMember.md&data=04%7C01%7CHerbert.Mauerer%40microsoft.com%7C1714a42b5893431742cd08d8774c1340%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637390515936011185%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Ee8Vs%2FxI%2BuhDfIhXEvEgdcoDWRlGTN13wzhtmBSdgAw%3D&reserved=0>

  *   Line 0, Column 0: [Warning-ms-prod-technology-invalid<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Freview.docs.microsoft.com%2Fhelp%2Fcontribute%2Fvalidation-ref%2Fms-prod-technology-invalid%3Fbranch%3Dmain&data=04%7C01%7CHerbert.Mauerer%40microsoft.com%7C1714a42b5893431742cd08d8774c1340%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637390515936011185%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=u%2BtzZl%2FfqoggftX4zC25LyEoYs0iBs%2BKh7hzjN6PMOk%3D&reserved=0>] Invalid value for 'ms.technology': 'powershell-windows' is not valid with 'ms.prod' value 'w10'.
  *   Line 0, Column 0: [Warning-ms-author-invalid<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Freview.docs.microsoft.com%2Fen-us%2Fhelp%2Fcontribute%2Fvalidation-ref%2Fms-author-invalid%3Fbranch%3Dmain&data=04%7C01%7CHerbert.Mauerer%40microsoft.com%7C1714a42b5893431742cd08d8774c1340%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637390515936021140%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=EM3%2Fr%2BbqYkbG2ImvJ6fL8tOGbbPas4fALCOhL%2FemOHU%3D&reserved=0>] Invalid value for 'ms.author', 'v-anbarr' is not a valid Microsoft alias.
Herbert Mauerer
[cid:image001.png@01D6CFA4.3EE157F0]<https://campus.partners.extranet.microsoft.com/sites/campus/Pages/wwcommunities.aspx>
Senior Escalation Engineer
Customer Services and Support EMEA
WW Communities Debugging<http://aka.ms/wwcomdebug>, SME
Which WW Community should you join? Click here<https://microsoft.sharepoint.com/teams/campusWWCs/Pages/Landing.aspx>
[cid:image002.jpg@01D6CFA4.3EE157F0]
Do you have feedback for me? Please ask me for a connect survey.

File: docset/windows/addsadministration/Remove-ADGroupMember.md (hunk 1 above)
Comment (HerbertMauerer) [reply docset/windows/addsadministration/Remove-ADGroupMember.md:4]: The original vendor account was not valid anymore. I tried different users I saw as contributing to this area of DOCS.
You can use the best fit for your purposes...

Comment (get-itips) [inline docset/windows/addsadministration/Remove-ADGroupMember.md:4]:
```suggestion
ms.author: v-kaunu
```

Comment (get-itips) [reply docset/windows/addsadministration/Remove-ADGroupMember.md:4]: Let's try to put v-kaunu ( @kateyanne ) again. Thank you @HerbertMauerer

**Comment by e0i** [comment]
Closing and re-opening to refresh validation checks.

**Comment by HerbertMauerer** [comment]
What is happening here?
note I also submitted 2175 trying to get the change done, it is also stuck.

**Comment by dariomws** [comment]
Hi @HerbertMauerer,

Could you kindly review the changes suggested by @get-itips? The alias must be "v-kaunu".

Thanks!

**Comment by HerbertMauerer** [comment]
thanks. I updated ms.author to v-kaunu.
I hope this progresses now.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by e0i** [comment]
@HerbertMauerer Thank you for your contributions and following up with the suggestions. The PR should be merged within a few hours.

**Pull request merged**